Country.json

```diff
@@ -0,0 +1,7 @@
+{
+ "country": "Ukraine",
+ "capital": "Kiev", 
+ "territory_km2": 603628, 
```

Comment:
I like how you mentioned the measurement unit!
Otherwise, it could've been ambiguous, but now we know it's sq. km, good job!